CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## Unreleased
+
+## 1.0.0
```

Comment:
The last comment from me :)
1. Add text into Unreleased section
2. Add the date of the first release

```suggestion
## Unreleased

*No changes*

## 1.0.0 - 2024.05.08
```